python/balance/controller.py

```diff
@@ -79,25 +107,21 @@ def credits_all(start=None, end=None):
 
 @balance.route('/cron/user-position/<string:period>', methods=['GET'], strict_slashes=False)
 def user_position(period):
-    users = User.query().filter(User.money < -0.01 and User.active == True).fetch()
+    users = User.query().filter(User.money < 0.0).filter(User.active == True).fetch()
     users_email_list = []
-
     usersJson = 'email;valor;active \n'
 
-    for u in users:
-        usersJson += str(u.email)+';'+str("%.2f" % round(u.money,2))+';'+str(u.active)+' \n'
-        users_email_list.append(str(u.email))
-
+    for idx,user in enumerate(users):
+        usersJson += str(user.email)+';'+str("%.2f" % round(user.money,2))+';'+str(user.active)+' \n'
```

Comment:
`usersCsv`? just a suggestion but you can comment if you disagree and why (I'm not that familiar with the code)